.kitchen.yml

```diff
@@ -39,6 +39,13 @@ suites:
     run_list:
       - recipe[influxdb-test::default]
     attributes:
+  - name: file
```

Comment:
awesome.  If you can, can you make a serverspec test case that tests if your version you selected is actually installed?

You should copy one `config_change` from this https://github.com/bdangit/chef-influxdb/tree/master/test/integration and call it `file`.

And tailor the serverspec tests for this usecase.  Let me know if you need help.